data/rulesets.js

```diff
@@ -717,6 +717,14 @@ let BattleFormats = {
 	natdex: {
 		effectType: 'Rule',
 		name: 'NatDex',
+		onValidateSet(set) {
+			// Item's other than Mega Stones, Z-Crystals, and the Orbs should still be illegal.
+			if (!set.item) return;
+			let item = this.dex.getItem(set.item);
+			if (item.isNonstandard === 'Past' && !item.megaStone && !item.zMove && !['redorb', 'blueorb'].includes(item.id)) {
```

Comment:
```suggestion
			if (item.isNonstandard === 'Past' && !item.zMove && !item.itemUser) {
```

(`item.itemUser` is a new field I'm introducing to the item database)